src/operation/get-parameters-as-json-schema.js

```diff
@@ -177,19 +177,21 @@ function searchForExampleByPointer(pointer, examples = []) {
  * @link https://json-schema.org/draft/2019-09/json-schema-validation.html
  * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md
  * @param {Object} data
- * @param {Object[]} prevSchemas
- * @param {String} currentLocation
- * @param {Object} globalDefaults
- * @param {Boolean} isPolymorphicAllOfChild
+ * @param {Object} opts
+ * @param {Object[]} opts.prevSchemas - Array of parent schemas to utilize when attempting to path together examples.
+ * @param {String} opts.currentLocation - Current location within the schema -- this is a JSON pointer.
+ * @param {Object} opts.globalDefaults - Object containing a global set of defaults that we should apply to schemas that match it.
+ * @param {Boolean} opts.isPolymorphicAllOfChild - Is this schema the child of a polymorphic `allOf` schema?
  */
-function constructSchema(
-  data,
-  prevSchemas = [],
-  currentLocation = '',
-  globalDefaults,
-  isPolymorphicAllOfChild = false
-) {
+function constructSchema(data, opts = {}) {
```

Comment:
Reworked this argument structure into an `opts` object because the arguments list was getting a little out of control with half of them being optional.